src/OpenTelemetry.Exporter.ZPages/ZPagesExporter.cs

```diff
@@ -34,11 +42,22 @@ public class ZPagesExporter : SpanExporter
         public ZPagesExporter(ZPagesExporterOptions options)
         {
             this.Options = options;
+
+            // Create a timer with one minute interval
+            this.minuteTimer = new Timer(60000);
+            this.minuteTimer.Elapsed += new ElapsedEventHandler(ZPagesSpans.TriggerMinuteCalculations);
```

Comment:
In a specification the only action here is Purge of stale data. Should we call it Purge than?